fledging.qmd

```diff
@@ -6,7 +6,76 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
 and want to continue their adventures in cloud computing, have the knowledge
 and resources to develop their own solutions.
 
-Here is a [presentation on fledging](https://docs.google.com/presentation/d/1-TiPN8bfY6iDL5EVcEuCQSjqFMs3jPSX2MlNWA6no2E/edit#slide=id.g2ece79fddf8_3_73) given at the Earth Science Information Partners
-(ESIP) conference in July, 2024.
+When thinking about cloud computing, we consider two components: where is our 
+data? And where is our compute? In general, if our data is already on our local
+computer, we will work with that data on our computer, so the scope we are 
+considering is how to work with *data that is hosted in the cloud*.
 
-More resources coming soon...
+![](images/data-compute-cloud-grid.png){width="400" fig-align="center"}
+
+It is also important to note the environmental costs of cloud computing - 
+it is [carbon-intensive](https://mit-serc.pubpub.org/pub/the-cloud-is-material/release/2), and so it is worth taking this into consideration when
```

Comment:
This is an important but complex topic, I don't want to overreach...

Reply:
See discussion [here](https://openscapes.slack.com/archives/CHZ2SMN7J/p1733966801974699)